flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CompletedCheckpointStoreTest.java

```diff
@@ -162,16 +160,16 @@ public void testGetAllCheckpoints() throws Exception {
 
         List<CompletedCheckpoint> actual = checkpoints.getAllCheckpoints();
 
-        assertEquals(expected.length, actual.size());
+        assertThat(actual).hasSameSizeAs(expected);
 
         for (int i = 0; i < expected.length; i++) {
-            assertEquals(expected[i], actual.get(i));
+            assertThat(actual.get(i)).isEqualTo(expected[i]);
```

Comment:
```java
assertThat(actual).containsExactly(expected);
```

Reply:
Thanks @Jiabao-Sun for the quick review, updated!